doozer/doozerlib/backend/konflux_olm_bundler.py

```
@@ -366,139 +467,209 @@ def _get_digest_image_pattern():
-        image_info_tasks = []
-        for match in matches:
+        art_references = {}
+        image_info_coros = []
+        for match in pattern.finditer(content):
             pullspec = match.group(0)
             namespace, image_short_name = match.group(1).rsplit('/', maxsplit=1)
             image_tag = match.group(2)
             art_references[pullspec] = (namespace, image_short_name, image_tag)
 
-        # Get image infos for ART-built images
         for pullspec, (namespace, image_short_name, image_tag) in art_references.items():
```

Ruff (0.15.21)

[warning] Loop control variable `pullspec` not used within loop body

(B007)